runtime/vm/LayoutFFITypeHelpers.hpp

```diff
@@ -42,13 +42,14 @@ class LayoutFFITypeHelpers
 	/* Function members */
 public:
 	LayoutFFITypeHelpers(J9VMThread *currentThread)
-			: _currentThread(currentThread)
-			, _vm(_currentThread->javaVM)
-	{ };
+					: _currentThread(currentThread)
+					, _vm(currentThread->javaVM)
+	{
+	}
 
 	/**
 	 * @brief Convert the preceding integer of a layout string from string to UDATA,
-	 * and put layout at the position after the integer
+	 * and put layout at the position after the integer.
 	 *
 	 * @param layout[in] A pointer to a c string describing the types of the struct elements. For example:
 	 * 		If layout is "16#2[#2[II]#2[II]]", it returns 16 in byte
```

Comment:
I don't think "in byte" makes sense here, and the period is missing.

Reply:
Rephrased the comment there to explain the returned value.